src/main/java/com/quirko/gui/GuiController.java

```diff
@@ -99,7 +117,23 @@ public void handle(KeyEvent keyEvent) {
                 if (keyEvent.getCode() == KeyCode.P) {
                     pauseButton.selectedProperty().setValue(!pauseButton.selectedProperty().getValue());
                 }
-
+                //different acculator for each difficulty level

```

Comment:
what is `acculator`?

Reply:
Hocam acceleration demek istemiştim o an yanlış hatırlamışım. Siz başlangıçta sabit hızla başlasınlar ama oyun ilerledikçe her zorluk seviyesi için farklı ivmelerle hızı arttırabilirsin diye bir fikir önerisi vermiştiniz. Ben de bu şekilde düşünerek farklı ivmeler tanımladım ve belli bir seviyeye kadar oyun boyunca hızı arttırdım. Belirli bir hıza ulaştığında hız sabit kaldı.